src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1002,6 +1046,27 @@ LedgerManagerImpl::applyTransactions(std::vector<TransactionFramePtr>& txs,
         tx->storeTransaction(mApp.getDatabase(), ledgerSeq, tm, ++index,
                              txResultSet);
     }
+
+    logTxApplyMetrics(ltx, numTxs, numOps);
+}
+
+void
+LedgerManagerImpl::logTxApplyMetrics(AbstractLedgerTxn& ltx, size_t numTxs,
+                                     size_t numOps)
+{
+    auto ledgerSeq = ltx.loadHeader().current().ledgerSeq;
+
+    CLOG(DEBUG, "Ledger")
+        << "Ledger: " << ledgerSeq << " txs: " << numTxs << ", ops: " << numOps
+        << ", AVG application time per tx: " << mTransactionApply.mean()
+        << " ± " << mTransactionApply.std_dev()
+        << ", MIN application time per tx: " << mTransactionApply.min()
+        << ", MAX " << mTransactionApply.max() << ", 75th percentile: "
+        << mTransactionApply.GetSnapshot().get75thPercentile();
+
+    CLOG(DEBUG, "Ledger") << "Ledger: " << ledgerSeq << " txs: " << numTxs
```

Comment:
Should we add the rate as a metric?

Reply:
Sure, added a counter to track the hit rate.